web/html/src/manager/virtualization/guests/list/guests-list.js

```diff
@@ -36,134 +38,137 @@ export function GuestsList(props: Props) {
+                  }
+                  if (row.subscribable) {
+                    return <a href={`/rhn/channels/ChannelDetail.do?cid=${row.channelId}`}>{row.channelLabels}</a>;
+                  }
+                  return row.channelLabels;
+                }}
+              />
+            ];
 
-          const actionsProvider =
-            (row) => {
-              if (props.foreignEntitled) {
-                return [];
-              }
-              const state = row.stateLabel;
-              return (
-                <div className="btn-group">
-                  {state !== 'running' && row.name !== 'Domain-0'
-                   && (
-                     <AsyncButton
-                       defaultType="btn-default btn-sm"
-                       title={t(state === 'paused' ? 'Resume' : 'Start')}
-                       icon="fa-play"
-                       action={() => onAction('start', [row.uuid], {})}
-                     />) }
-                  {state === 'running' && row.name !== 'Domain-0'
-                   && createModalButton('suspend', modalsData, row) }
-                  {state !== 'stopped' && row.name !== 'Domain-0'
-                   && createModalButton('shutdown', modalsData, row) }
-                  {(state === 'paused' || state === 'running') && createModalButton('restart', modalsData, row) }
-                  {props.saltEntitled && state === 'running' && (
+            const actionsProvider =
+              (row) => {
+                if (props.foreignEntitled) {
+                  return [];
+                }
+                const state = row.stateLabel;
+                return (
+                  <div className="btn-group">
+                    {state !== 'running' && row.name !== 'Domain-0'
+                     && (
+                       <AsyncButton
+                         defaultType="btn-default btn-sm"
+                         title={t(state === 'paused' ? 'Resume' : 'Start')}
+                         icon="fa-play"
+                         action={() => onAction('start', [row.uuid], {})}
+                       />) }
+                    {state === 'running' && row.name !== 'Domain-0'
+                     && createModalButton('suspend', modalsData, row) }
+                    {state !== 'stopped' && row.name !== 'Domain-0'
+                     && createModalButton('shutdown', modalsData, row) }
+                    {(state === 'paused' || state === 'running') && createModalButton('restart', modalsData, row) }
+                    {props.saltEntitled && state === 'running' && (
+                      <LinkButton
+                        title={t('Graphical Console')}
+                        className="btn-default btn-sm"
+                        icon="fa-desktop"
+                        href={`/rhn/manager/systems/details/virtualization/guests/${props.serverId}/console/${row.uuid}`}
+                        target="_blank"
+                      />
+                    )}
```

Comment:
This indentation/code-style is driving me crazy sorry :sweat_smile:  

Could you please rewrite with some empty line between objects, and whenever it is possible without breaking lines? Something like the following:

```
...

{state === 'running' && row.name !== 'Domain-0' && createModalButton('suspend', modalsData, row) }

{state !== 'stopped' && row.name !== 'Domain-0' && createModalButton('shutdown', modalsData, row) }

{(state === 'paused' || state === 'running') && createModalButton('restart', modalsData, row) }

...
```